mesh/node.c

```diff
@@ -1614,14 +1655,22 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 
 		cb(req->user_data, MESH_ERROR_FAILED, node);
 	} else {
-		/* Handle failed Join and Create requests */
+		/* Handle failed Join, Create and Import requests */
 		if (node)
 			node_remove(node);
 
 		if (req->type == REQUEST_TYPE_JOIN) {
 			node_join_ready_func_t cb = req->cb;
 
 			cb(NULL, NULL);
+
+		} else if (req->type == REQUEST_TYPE_IMPORT) {
+			struct node_import_request *import_data =
+								req->user_data;
+			node_ready_func_t cb = req->cb;
+
+			cb(import_data->user_data, MESH_ERROR_FAILED, NULL);
```

Comment:
Other request types call the callback with `req->user_data`, please keep this consistent here.

Reply:
Won't be done :)